pandas/tests/io/parser/test_check_for_bom.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from pandas import read_csv
+
+
+def test_first_row_bom():
```

Comment:
Can you move this to `parser/test_common.py` and use the `all_parsers` fixture?

Reply:
Yes, certainly. That makes sense.